protocol-units/settlement/mcr/manager/Cargo.toml

```diff
@@ -30,6 +30,10 @@ mcr-settlement-client = { workspace = true, features = ["mock"] }
 [features]
 default = ["stub"]
 stub = []
```

Comment:
Might as well remove this feature, it does not seem to be used.

Reply:
Removed

Reply:
It's still there in the latest diff?